src/Interpreters/Streaming/Aggregator.h

```diff
@@ -834,10 +836,12 @@ class Aggregator final
     friend class AggregatingInOrderTransform;
 
     /// proton: starts
+    friend struct AggregatingHelper;
```

Comment:
Why use struct? Is it better to just call the functions directly?

Reply:
In AggregatingHelper, almost methods invoked private methods of Aggregator, so here use a `friend struct` instead of `a lot of friend functions`